orchestration/dags/data_gcp_dbt/models/mart/native/mrt_native__consultation.sql

```diff
@@ -0,0 +1,52 @@
+{{
+    config(
+        **custom_incremental_config(
+        incremental_strategy = 'insert_overwrite',
+        partition_by = {'field': 'consultation_date', 'data_type': 'date'}
+    )
+) }}
+
+with discoveries_by_consultation as (
+SELECT consultation_id,
+    MAX(CASE WHEN type = 'item' and consulted_entity is not null then 1 else 0 END) AS item_score,
```

Comment:
Pourquoi consulted_entity is not null est nécessaire ici ? La colonne est pas toujours peuplée ?

Reply:
You're right ! On peut retirer la clause en effet, car si on a un type = 'item', ça veut dire qu'on aura forcément une consulted_entity. Tu peux commit ? :)